src/Nethermind/Nethermind.Consensus/Validators/BlockValidator.cs

```diff
@@ -46,6 +46,30 @@ public bool Validate(BlockHeader header, bool isUncle)
         return _headerValidator.Validate(header, isUncle);
     }
 
+
+    /// <summary>
+    /// Orphaned block validation applies to blocks without parent which may happen during BeaconSync
+    /// </summary>
+    /// <param name="block">A block to validate</param>
+    /// <param name="error">Error description in case of failed validation</param>
+    /// <returns>Validation result</returns>
+    public bool ValidateOrphanedBlock(Block block, out string? error)
+    {
+        if (_specProvider.GetSpec(block.Header).IsEip4844Enabled)
+        {
+            ulong calculated = BlobGasCalculator.CalculateBlobGas(block.Transactions);
+            if (calculated != block.BlobGasUsed)
+            {
+                error = $"Invalid {nameof(block.BlobGasUsed)}: {block.BlobGasUsed}, expected {calculated}";
+                if (_logger.IsDebug) _logger.Debug($"{Invalid(block)} {error}");
+                return false;
+            }
+        }
```

Comment:
Can we extract it from here and `ValidateEip4844Fields` to single method?

Reply:
Replaced it with `ValidateEip4844Fields`. We can actually use more things from `ValidateSuggestedBlock` like `ValidateTransactions`/`ValidateUnclesHashMatches`/etc as they do not depend on existence of parent. Wdyt guys? cc: @MarekM25

Reply:
yes, yes :) we can do it in a separate PR - agreed